roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -72,6 +72,16 @@ def get_var(task_vars, *keys, **kwargs):
         raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
     return value
 
+def normalized_release(task_vars):
+    """Helper function to normalize the release format to dotted numbers"""
+    release = str(get_var(task_vars, "openshift_release"))
+    if release.startswith('v'):
+        release = release[1:]  # v3.3 => 3.3
+    return release
+
+def normalized_minor_release(task_vars):
```

Comment:
I thought about having the normalizers be simple string operations but then every invocation would be

    normalized_minor_release(get_var(task_vars, "openshift_release"))

and I hate to see that kind of repetition. That's why I write methods :) Though obviously encapsulation is important.

Actually I don't like passing around `task_vars` everywhere to begin with. Why not just pass it to the check constructor to be put in an attribute and used later?